plugins/spotify/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/spotify/MusicResource.java

```diff
@@ -168,103 +171,189 @@ public String setDevice(@PathParam("deviceValue") String deviceValue) {
 	}
 
 	/**
-	 * this method allows to search artist, track, playlist and album names
+	 * this method allows to search for tracks
 	 * 
 	 * @param searchText
 	 *            the text you want to search
-	 * @param type
-	 *            artist, track, playlist, album
 	 * @param limit
 	 *            how many results maximal searched for
-	 * @return one output String with all results
+	 * @return a array with TrackEntities
 	 */
 	@POST
```

Comment:
Why are the four search endpoints HTTP POST and not GET? Is there a reason behind it?